lib/index.js

```diff
@@ -189,6 +188,27 @@ module.exports.render = function(options, cb) {
     };
   }
 
+  if (options.functions) {
+    Object.keys(options.functions).forEach(function(signature) {
+      var callback = options.functions[signature];
+
+      options.functions[signature] = function() {
+        var args = Array.prototype.slice.call(arguments),
+            bridge = args.pop();
+
+        function done(data) {
+          bridge.success(data);
+        }
+
+        var result = callback.apply(this, args.concat(done));
```

Comment:
Yes this is what I just told @matryo on gitter chat :)
This PR is experimental. the real meat is still #644. :wink: